static/app/components/events/autofix/autofixSolution.tsx

```diff
@@ -474,6 +473,8 @@ function AutofixSolutionDisplay({
   const containerRef = useRef<HTMLDivElement>(null);
   const iconFixRef = useRef<HTMLDivElement>(null);
 
+  const changesDisabled = repos.every(repo => repo.is_readable === false);
```

Comment:
shouldn't it be disabled if _any_ repo is not accessible? (I can see an argument both ways I guess, so up to you)

Reply:
you could have the case for example, that the fix is in seer, but you have the sentry repo leftover as context that isn't readable...ideally we can just pinpoint which repos the solution is in to make this condition better.

I should also update the guard message here to make it more clear...

Reply:
the solution timeline does include relevant repo names in its payload, so we could use that, but might not be reliable and might not work with human instructions

Reply:
yeah right now it's inconsistent, we should update the solution formatter to output repo names/ids for each step, for now I guess this should be sufficient